main.sh

```diff
@@ -188,23 +206,32 @@ run_daos_test(){
 #Run cart self_test
 run_cart_test(){
     echo -e "\nCMD: Starting CaRT self_test...\n"
-    END_POINTS="-`expr $DAOS_SERVERS - 1`"
-    for rpc in "${SELF_TEST_RPC[@]}"; do
-        for point in 0 1; do
-                cmd="orterun --timeout 3600 $PSM2_CLIENT_PARAM -np 1 self_test
-		--group-name daos_server --endpoint 0-$point:0 --master-endpoint 0$END_POINTS:0
-	        --message-sizes 'b1048576',' b1048576 0','0 b1048576',' b1048576 i2048',' i2048 b1048576',' i2048',' i2048 0','0 i2048','0' 
-        	--max-inflight-rpcs $rpc --repetitions 100"
-	        echo $cmd
-		echo
-        	eval $cmd
-	        if [ $? -ne 0 ]; then
-        	    echo -e "\nSTATUS: CART self_test FAIL\n"
-        	    exit 1
-	        else
-        	    echo -e "\nSTATUS: CART self_test SUCCESS\n"
-	        fi
-        done
+
+    while [  ${ST_MIN_SRV} -le ${ST_MAX_SRV} ]; do
+	let last_srv_index=$(( ${ST_MIN_SRV}-1 ))
+
+        for max_inflight in "${ST_MAX_INFLIGHT[@]}"; do
+            cmd="orterun --timeout ${ST_TIMEOUT} -np 1 ${VERBS_CLIENT_PARAM} 
+                 --map-by node  --hostfile Log/${SLURM_JOB_ID}/daos_client_hostlist
+	         --output-filename testLogs
+	         -x PATH -x LD_LIBRARY_PATH ${ST_PARAMS}
```

Comment:
So -x D_LOG_FILE=testLogs/output.log is part of ST_PARAMS, will be the same for each cmd you run in while loop and for max_inflight too?

Reply:
ah yes, will update to split them up.